ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/searches/searches.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+import Search from '../search'
+import Text from '../../container/input-wrappers/text'
+import Dropdown from '../../presentation/dropdown'
+import NavigationBehavior from '../../presentation/navigation-behavior'
+import MenuSelection from '../../presentation/menu-selection'
+
+const SearchesPage = styled.div`
+  height: 100%;
+  overflow-y: auto;
+
+  background-color: ${props => props.theme.backgroundContent};
+`
+
+const ContextBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  background-color: ${props => props.theme.backgroundAccentContent};
+
+  .filter-searches {
+    width: calc(300px + (2 * ${props => props.theme.largeSpacing}));
```

Comment:
❓ What is this `300px` value based on? Does it scale well when the user changes the theme spacing?